storybook/stories/Layout/AppText.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { AppText } from "../../../src/components/Layout/AppText";
+import { View, Text } from "react-native";
+import { size, color } from "../../../src/common/styles";
+
+const appTextElement = (): JSX.Element[] => {
+  return [
+    <>
+      <AppText>
+        <Text>Default app text</Text>
+      </AppText>
+    </>,
+    <>
+      <AppText style={{ color: color("red", 60) }}>
+        <Text>customize app text in (color(&quot;red&quot;, 60))</Text>
+      </AppText>
+    </>,
+    <>
+      <AppText style={{ fontSize: size(3) }}>
+        <Text>customize app text in fontSize: size(3)</Text>
```

Comment:
"Customised text with fontSize-3".
Please use `fontSize` and not `size`, they are different

Reply:
@maksha19 this is not resolved yet